.github/workflows/link_reminder.yml

```diff
@@ -0,0 +1,33 @@
+name: Link reminder
+
+on:
+  pull_request:
+    types: [opened, reopened]
```

Comment:
you probably shouldn't run the workflow on PR reopens, as it would then post a duplicate comment